### ISSUE TITLE
Can't pickle local object 'merge_configuration_file' after upgrading from 1.6.1 to 1.7.2

### ISSUE THREAD

**Issue by nicoddemus** [PyCQA/autoflake#168]
Hi,

Our automatic pre-commit [autoupdate job](https://github.com/pytest-dev/pytest/pull/10368) has upgraded `autoflake` from 1.6.1 to 1.7.2, and now we are getting [this error](https://results.pre-commit.ci/run/github/37489525/1665485632.FdiwDQkfRPikghjLAeHb7A):

```
autoflake................................................................Failed
- hook id: autoflake
- exit code: 1

Traceback (most recent call last):
  File "/pc/clone/XOD6b1sUTx-B7vO1RWDKXA/py_env-3.10/bin/autoflake", line 8, in <module>
    sys.exit(main())
  File "/pc/clone/XOD6b1sUTx-B7vO1RWDKXA/py_env-3.10/lib/python3.10/site-packages/autoflake.py", line 1519, in main
    return _main(
  File "/pc/clone/XOD6b1sUTx-B7vO1RWDKXA/py_env-3.10/lib/python3.10/site-packages/autoflake.py", line 1501, in _main
    exit_status |= fut.get()
  File "/usr/lib/python3.10/multiprocessing/pool.py", line 771, in get
    raise self._value
  File "/usr/lib/python3.10/multiprocessing/pool.py", line 537, in _handle_tasks
    put(task)
  File "/usr/lib/python3.10/multiprocessing/connection.py", line 211, in send
    self._send_bytes(_ForkingPickler.dumps(obj))
  File "/usr/lib/python3.10/multiprocessing/reduction.py", line 51, in dumps
    cls(buf, protocol).dump(obj)
AttributeError: Can't pickle local object 'merge_configuration_file.<locals>.<lambda>'
```

At first I thought it might be some flaky problem, but got the same error after running it again.

Any hints?

**Comment by fsouza** [comment]
Hey, should be fixed now in 1.7.3. Can you re-run `pre-commit autoupdate`?

**Comment by nicoddemus** [comment]
That was quick. :grin: 

That fixed it. Much appreciated, thanks!
